CHANGES.md

```diff
@@ -1,3 +1,29 @@
+
+@@ -1,3 +1,27 @@
+## 1.7.1 2019-12-18
+Linkerd 1.7.1 adds a parameter named `initialStreamWindowSizeKB` to 
+MeshInterpreterInitializer.scala. This parameter is used to configure the 
+[initial flow-control window size](https://http2.github.io/http2-spec/#InitialWindowSize)
+that the H2 client in the `io.l5d.mesh` interpreter sends to the H2 server. 
```

Comment:
We've also greatly increase the default value to allow the mesh interpreter to receive much larger responses from Namerd.